### PR TITLE
Text: Adapt to Akka 2.6

### LINKED ISSUE #2503: text: Akka 2.6 features
- [ ] Backports: If a `RetryFlow(Coordinator)` is used, remove it and use the Akka implementation
- [ ] Replace `Source.setup` / `Flow.setup` with `Source.fromMaterializer` / `Flow.fromMaterializer` 
- [ ] Tests: Use `ActorSystem` instead of `Materializer` when creating a stream
- [ ] Resolve other Akka deprecations
- [ ] Enable `fatalWarnings` for the connector
- [ ] Docs: Remove references about the `Materializer` from the text

(This is a bulk-created issue to manage updating Alpakka for the 3.0 release. Not all items apply to all connectors.)

### PR DESCRIPTION
**Pull request by ennru**
Fixes #2503 

(doesn't enable fatal warnings as that triggers a total rebuild of Alpakka, I'll spare that until we have merged a few)﻿

### CODE REVIEW

File: text/src/test/scala/akka/stream/alpakka/text/scaladsl/CharsetCodingFlowsSpec.scala [hunk 1]
```diff
@@ -77,7 +74,7 @@ class CharsetCodingFlowsSpec
           .via(TextFlow.encoding(StandardCharsets.US_ASCII))
           .intersperse(ByteString("\n"))
           .runWith(FileIO.toPath(targetFile))
-      result.futureValue.status should be(Success(Done))
+      result.futureValue.count should be > 50L
```
Comment (seglo) [inline text/src/test/scala/akka/stream/alpakka/text/scaladsl/CharsetCodingFlowsSpec.scala:77]: What's the purpose of the `50L` constant? When I ran this test myself the count was `7430`.

**Review by seglo — APPROVED** [review]
LGTM. Question about the constants being used.

File: text/src/test/scala/akka/stream/alpakka/text/scaladsl/CharsetCodingFlowsSpec.scala (hunk 1 above)
Comment (ennru) [reply text/src/test/scala/akka/stream/alpakka/text/scaladsl/CharsetCodingFlowsSpec.scala:77]: The input is system dependent and not stable.

Comment (seglo) [reply text/src/test/scala/akka/stream/alpakka/text/scaladsl/CharsetCodingFlowsSpec.scala:77]: Wouldn't it be safer to rely on `status` then?  How did you arrive at `50`?

Comment (ennru) [reply text/src/test/scala/akka/stream/alpakka/text/scaladsl/CharsetCodingFlowsSpec.scala:77]: That's where this began: `status` is deprecated in Akka 2.6. If the future completes status is always `Success(Done)`...
To add some other validation I randomly picked 50 to show that some data came through. It really doesn't matter.

**Pull request merged**